convert2rhel/systeminfo.py

```diff
@@ -138,21 +138,38 @@ def _get_gpg_key_fingerprints(self):
     def _get_pkg_blacklist(self):
         return self._get_cfg_opt("pkg_blacklist").split()
 
-    def _generate_rpm_va(self):
+    def _generate_rpm_va(self, log_filename='rpm_va.log'):
         """Verify the installed files on the system with the rpm metadata
         for debug and support purposes."""
         if tool_opts.no_rpm_va:
             self.logger.info("Skipping execution of 'rpm -Va'.")
-            return
+            return False
 
         self.logger.info("Running the 'rpm -Va' command which can take several"
                          " minutes. It can be disabled by using the"
                          " --no-rpm-va option.")
         rpm_va, _ = utils.run_subprocess("rpm -Va", print_output=False)
-        output_file = os.path.join(logger.LOG_DIR, "rpm_va.log")
+        output_file = os.path.join(logger.LOG_DIR, log_filename)
         utils.store_content_to_file(output_file, rpm_va)
-        self.logger.info("The 'rpm -Va' output has been stored in the %s file"
-                         % output_file)
+        self.logger.info("The 'rpm -Va' output has been stored in the %s file" % output_file)
+
+        return True
+
+    def generate_modified_rpms(self):
+        """Generate a log and print stdout message with modified rpms after the convert process."""
+        if not self._generate_rpm_va(log_filename='rpm_va_after_conversion.log'):
+            self.logger.info("Unable to run 'rpm -Va' after the conversion.")
+            return
+
+        before_rpm_va = utils.get_file_content(os.path.join(logger.LOG_DIR, "rpm_va.log"), True)
+        rpm_va = utils.get_file_content(os.path.join(logger.LOG_DIR, "rpm_va_after_conversion.log"), True)
+        diff_packages_list = list(set(before_rpm_va).symmetric_difference(set(rpm_va)))
+
+        if diff_packages_list:
+            self.logger.info("INFO - List of modified rpm after the conversion process:")
+            for pkg in diff_packages_list:
+                self.logger.info(" - %s" % pkg)
```

Comment:
```suggestion
        if modified_rpm_files_diff:
            self.logger.info("Comparison of modified rpm files from before and after the conversion:\n%s" % modified_rpm_files_diff)
```
The prefix, similar to INFO, is being added by the logger formatter.